l10n_it_fatturapa_pec/data/sdi_channel_demo.xml

```diff
@@ -3,6 +3,7 @@
      License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
 <odoo>
     <record id="sdi_channel_pec" model="sdi.channel">
```

Comment:
Lo spostamento di questo file causa il seguente problema:

1. Prendere in checkout `14.0`
3. Creare un DB dove:
5. installare fatturapa_pec
7. creare un canale SdI con i server mail in entrata e in uscita
9. associare il canale all'azienda
11. Stoppare Odoo e prendere in checkout https://github.com/OCA/l10n-italy/pull/2708
13. Far ripartire Odoo aggiornando all

**Comportamento attuale**
Il canale non è più collegato all'azienda.
I server di posta non sono più collegati al canale SdI.

**Comportamento atteso**
Nessuna modifica alle configurazioni.

Per correggere, ti ho proposto https://github.com/GSLabIt/l10n-italy/pull/1, puoi verificare se hai lo stesso problema e se la PR risolve?
Ho fatto la stessa modifica in #2810 e per me risolve.

Reply:
Grazie, ho fatto un push

Reply:
:+1: per le prossime volte, ti sarebbe probabilmente venuto più comodo fare il merge (rebase and merge) della PR che avevo proposto: così avremmo anche mantenuto l'authorship sulle modifiche.
Per questa volta, visto che sono 5 righe senza una particolare logica, per me va bene così; ma per modifiche più complesse sarebbe corretto mantenere l'authorship dell'autore originale.